source/architecture/single_page.md

```diff
@@ -0,0 +1,42 @@
+# Build a single page site
+
+You can create a technical documentation site that contains all of its content in a single page.
+
+This is suitable for documentation sites that have do not have a lot of content, and do not need to have topics split into individual pages.
+
+## Organise content files 
+
+1. Create [link] the content files for your documentation site.
```

Comment:
Does '[link]' just imply that information is to be added later?

Reply:
yes